worlds/shivers/Rules.py

```diff
@@ -0,0 +1,253 @@
+import typing
+from collections.abc import Callable
+from BaseClasses import CollectionState
+from worlds.AutoWorld import World
+from worlds.generic.Rules import forbid_item
+from .Options import get_option_value
+
+
+def water_capturable(state: CollectionState, player: int) -> bool:
+    return (state.can_reach("Lobby", "Region", player) or (state.can_reach("Janitor Closet", "Region", player) and cloth_capturable(state, player))) \
+        and state.has("Water Pot Bottom", player) \
```

Comment:
`state.has_all`. same comment for most of this file

Reply:
Updated Rules.py to use state.has_all throughout